client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -174,38 +170,52 @@ h2.action-panel__title {
 }
 
 .email-provider-stacked-card__title-price-badge {
-	width: $width-right-panel-card;
+	width: 100%;
+	margin-left: 10px;
+	@include break-small {
+		width: $width-right-panel-card;
+	}
 }
 
 .email-provider-stacked-card__provider-form-and-right-panel {
 	display: none;
 }
 
+.email-providers-stacked-comparison__main.main .card.action-panel.promo-card.email-providers-stacked-comparison__provider-card .action-panel__figure {
```

Comment:
I think we need to override the padding for mobile, likely on  `.email-providers-stacked-comparison__main.main` -- per the designs I think it should be 20px, but it's at 24px at the moment. (And yes, those 8px do make a big difference at narrow widths with longer domain names.)

Reply:
Done in parent component